Elasticsearch: Remove browser access mode (#49014)
grafana/grafana: docs/sources/datasources/elasticsearch.md

> **Note:** If you're not seeing the `Data Sources` link in your side menu it means that your current user does not have the `Admin` role for the current organization.

| Name      | Description                                                                                                           |
| --------- | --------------------------------------------------------------------------------------------------------------------- |

> | `Name`    | The data source name. This is how you refer to the data source in panels and queries.                                 |

Reviewer: ```suggestion
| `Name`    | Data source name. This is how you refer to the data source in panels and queries.                                 |
```

> | `Default` | Default data source means that it will be pre-selected for new panels.                                                |

Reviewer: ```suggestion
| `Default` | Data source is pre-selected for new panels.                                                |
```

| `Url`     | The HTTP protocol, IP, and port of your Elasticsearch server.                                                         |

> | `Access`  | This setting is only here for historical reasons. Use "Server (default)", otherwise the datasource will not function. |

Reviewer: ```suggestion
| `Access`  | Do not use Access. Use "Server (default)" or the datasource won't function. |
```
